missions/ME_Austria_Missions.txt

```diff
@@ -1,1674 +1,6981 @@
 				}
+				add_mil_power = 100
 			}
-		}
-	}
-	
-	austria_control_constantinople = {
-		icon = mission_sea_battles
-		position = 12
-		required_missions = { austria_control_ragusa }
-		trigger = {
-			151 = {
-				has_most_province_trade_power = ROOT
+			else = {
+				change_mil = 1
 			}
-		}
-		effect = {
 			add_country_modifier = {
-				name = "austria_control_of_silk_road"
-				duration = 7300
+				name = me_hab_a_new_iron_emperor
+				duration = 9125
 			}
 		}
 	}
+}
+
+me_hab_1_hre = {
+	slot = 1
+	generic = no
+	ai = yes
+	potential = {
+		tag = HAB
+		has_country_flag = use_missions_expanded_missions_instead_of_paradox
+		has_country_flag = me_hab_german_path
+	}
+	has_country_shield = yes
 	
-	austria_global_trade = {
-		icon = mission_dominate_home_trade_node
-		position = 13
+	me_hab_salzburg = {
+		icon = mission_fortify_rajputana
 		required_missions = {
-			austria_control_constantinople
-			austria_build_manufactories
-			austria_colonial_empire
+			me_hab_swabia
 		}
+		position = 3
+		
 		trigger = {
-			home_trade_node = {
-				highest_value_trade_node = yes
-				trade_share = {
-					country = ROOT
-					share = 75
-				}
+			76 = {
+				owned_by = ROOT
 			}
 		}
+		
 		effect = {
-			add_years_of_income = 1
-			add_prestige_or_monarch_power = { amount = 50 }
+			76 = {
+				add_province_modifier = {
+					name = me_hab_castle_of_salt
+					duration = 7300
+				}
+			}
+			custom_tooltip = me_hab_salt_modifier_renew
 		}
 	}
 }
 
-ME_austria_missions_2 = {
-	slot = 2
+me_hab_1_1 = {
+	slot = 1
 	generic = no
 	ai = yes
 	potential = {
-		AND = {
-			tag = HAB
-			has_country_flag = use_missions_expanded_missions_instead_of_paradox
-		}
+		tag = HAB
+		has_country_flag = use_missions_expanded_missions_instead_of_paradox
 	}
 	has_country_shield = yes
-	
-	austria_decline_of_hungary = {
-		icon = me_mission_hungary_flag
-		position = 2
-		required_missions = { austria_control_electors }
-		trigger = {
+
+	me_hab_slovakia = {
+		icon = personal_diplomacy
+		required_missions = {
+			
+		}
+		position = 4
+		provinces_to_highlight = {
 			OR = {
-				AND = {
-					is_emperor = yes
-					marriage_with = HUN
+				province_id = 129
+				province_id = 4751
+			}
+			NOT = { country_or_non_sovereign_subject_holds = ROOT }
+		}
+		trigger = {
+			if = {
+				limit = {
+					129 = {
+						owned_by = CLI
+					}
+					4751 = {
+						owned_by = CLI
+					}
+				}
+				4751 = {
+					is_claim = ROOT
 				}
-				overlord_of = HUN
-				NOT = { exists = HUN }
 			}
-			OR = {
-				AND = {
-					total_development = HUN
-					total_development = 200
+			else = {
+				129 = {
+					owned_by = ROOT
 				}
-				AND = {
-					TUR = {
-						num_of_provinces_owned_or_owned_by_non_sovereign_subjects_with = {
-							value = 5
-							OR = {
-								area = transylvania_area
-								area = southern_transylvania_area
-								area = alfold_area
-							}
-							is_city = yes
-						}
-					}
-					total_development = HUN
+				4751 = {
```

Comment:
provinces to highlight/trigger discrepancy (country_or_non_sovereign_subject_holds vs more complicated trigger with owned_by)

Reply:
Leftovers, leftovers. I will let it for now